package.json

```diff
@@ -43,10 +43,12 @@
     "rechoir": "^0.6.2"
   },
   "devDependencies": {
+    "ava": "^0.16.0",
+    "coffee-script": "^1.10.0",
     "eslint": "^2.0.0",
     "eslint-config-airbnb-base": "^3.0.0",
     "eslint-plugin-import": "^1.11.1",
-    "coffee-script": "^1.10.0",
+    "jscodeshift": "^0.3.28",
```

Comment:
Ah, you're probably right. Thanks for pointing this out. I'll remove it and verify.